src/ocamlorg_frontend/pages/blog.eml

```diff
@@ -53,12 +53,12 @@ Layout.render
                             </div>
                         </div>
                         <% (match item.preview_image with | None -> %> 
-                            <div class="flex-none w-64">
-                                <a href="<%s Option.value ~default:(Url.blog_post item.source.id item.slug) item.url %>"><img class="w-full" src="<%s Ocamlorg_static.Asset.url "logo.svg" %>" alt="default logo"></a>
+                            <div class="flex justify-center items-center w-82  md:flex-none md:justify-start md:items-start md:w-64  ">
+                                <a href="<%s Option.value ~default:(Url.blog_post item.source.id item.slug) item.url %>" class="w-full"><img class="w-full object-cover" src="<%s Ocamlorg_static.Asset.url "logo.svg" %>" alt="default logo"></a>
                             </div>
                         <% | Some image -> %>
-                            <div class="flex-none w-64">
-                                <a href="<%s Option.value ~default:(Url.blog_post item.source.id item.slug) item.url %>"><img class="w-full" src="<%s image %>" alt=""></a>
+                            <div class="flex justify-center items-center w-82  md:flex-none md:justify-start md:items-start md:w-64  ">
```

Comment:
```suggestion
                            <div class="flex justify-center items-center w-full max-w-88 md:flex-none md:justify-start md:items-start md:w-64  ">
```

We're designing for screens of minimum size 320px. Setting `w-82` would be too much on that smallest screen size. Using `max-w-` to limit the maximum width that can be taken up would probably serve the intended purpose here.